### PR TITLE
Readme update for remote install

### PR DESCRIPTION
**Pull request by dannylamb**
This is how I ended up doing it.  Any insight from other folks is appreciated.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,9 +42,39 @@ If you are not using `vagrant up` to bring up a box, and are running `ansible-pl
...
 2. `vagrant up`
 
+### All-in-one provisioning with Ansible
+
+If you want to provision an all-in-one remote Ubuntu environment, like a production server:
+
+1. SSH into your remote server and add an `ubuntu` [user with sudo privileges](https://www.digitalocean.com/community/tutorials/how-to-create-a-sudo-user-on-ubuntu-quickstart)
+1. Clone the repository onto your local machine
+1. Create an inventory for your new environment ('production' in this example): `cp -r inventory/vagrant inventory/production`
+1. Edit `inventory/produciont/hosts` to point to your new environment by changing 'default' line to:
```
Comment (whikloj) [inline README.md]: typo `produciont`

**Comment by jonathangreen** [comment]
https://github.com/dannylamb/claw-playbook/pull/1

**Comment by Natkeeran** [comment]
@dannylamb The instructions do not say if an ubuntu user is needed in the target machine.  Is it needed?

**Comment by dannylamb** [comment]
@Natkeeran As far as I can tell, yes.  I tried with and without an ubuntu user and it totally flopped wihtout it.  What I'm not sure of is if we can get away with any user that has sudo privileges, or if it _must_ be named `ubuntu`.  I'll have to spin one up and try to know for sure.

**Comment by jonathangreen** [comment]
I think after my PR it can be any user with sudo privileges, but I have to go give it a try to make sure.

**Comment by Natkeeran** [comment]
@dannylamb @jonathangreen 
You need an ubuntu user because some roles/configuration expects that ubuntu home directly.  That user need not have sudo privileges, if that is not the ansible user.

**Comment by dannylamb** [comment]
@Natkeeran @jonathangreen 
```bash
daniel@daniel-Latitude-3560:~/Code/Environments/claw-playbook$ grep -rn "/home/ubuntu" .
./roles/internal/webserver-app/defaults/main.yml:7:webserver_app_jwt_key_path: /home/ubuntu/auth
./roles/internal/webserver-app/defaults/main.yml:8:webserver_app_jwt_config_path: /home/ubuntu/configs/jwt
./roles/internal/webserver-app/defaults/main.yml:10:webserver_app_drupal_config_path: /home/ubuntu/configs/drupal
./.vagrant/machines/default/virtualbox/synced_folders:1:{"virtualbox":{"/home/ubuntu/islandora":{"guestpath":"/home/ubuntu/islandora","hostpath":"/home/daniel/Code/Environments/claw-playbook","disabled":false,"__vagrantfile":true}}}
```
Looks like there's three spots where we have to specify the user directory.  I can either update this PR to add a 'home directory' variable or can update the docs to list these as configs that need to be changed.

File: README.md [hunk 2]
```diff
@@ -53,27 +53,26 @@ If you're looking for a development environment, using our Vagrant deployment is
...
+  1. You should modify `group_vars\all\passwords.yml` to use more secure passwords. These passwords can be encrypted using [Ansible Vault](https://docs.ansible.com/ansible/latest/vault.html) if you wish to keep your inventory secure.
+  1. Change the `drupal_trusted_host` configuration in `inventory/production/group_vars/webserver/drupal.yml` to reflect your IP or domain name
+  1. Change the Apache's port to 80 in `inventory/production/group_vars/webserver/apache.yml`
+  1. Any other variable changes you wish.
 1. Install the roles using `ansible-galaxy`: `$ ansible-galaxy install -r requirements.yml`
-1. Provision the server with `$ ansible-playbook -i inventory/production -e "islandora_distro=ubuntu/xenial64"`
+1. Provision the server with `$ ansible-playbook -i inventory/production`
+  - If the host you are provisioning is a Ubuntu 16.04 machine, you may wish to have the playbook install Python for you. This is a requirement to run the playbook. You can do this by passing an additional variable on the command line like this. `$ ansible-playbook -i inventory/production -e "islandora_distro=ubuntu/xenial64"`
```
Comment (Natkeeran) [inline README.md]: I think the command should be:`$ ansible-playbook -i inventory/production playbook.yml -e "islandora_distro=ubuntu/xenial64"`

**Comment by Natkeeran** [comment]
This https://github.com/Islandora-Devops/claw-playbook/pull/41 helps in remove the ubuntu user dependency.  But, it still leaves this one in https://github.com/Islandora-Devops/claw-playbook/blob/master/roles/internal/webserver-app/defaults/main.yml#L12.  Can use the www-data user.

Also, changed the port to 80.  Seems to index it in bigdata.  But, does not seem to sync to fedora!

**Comment by whikloj** [comment]
@dannylamb Could you rebase this please? @Natkeeran was able to test it and is ready to merge.

**Comment by dannylamb** [comment]
@whikloj @Natkeeran rebased

**Comment by dannylamb** [comment]
I must've clobbered the change with @Natkeeran's feedback.  Think I did it straight in the browser and didn't pull that down to my local before rebase/force pushing.

**Comment by dannylamb** [comment]
And apparantly @jonathangreen's pull as well...   I have no idea what I did wrong there.  Sry folks.  Should be good now.

**Comment by Natkeeran** [comment]
@whikloj @jonathangreen 
Tested this.  Seems good.

**Pull request merged**